src/app/area/areas-arborescence/areas-arborescence.page.html

```diff
@@ -0,0 +1,62 @@
+<ion-header>
+  <ion-toolbar>
+      <ion-buttons slot="start">
+          <ion-back-button></ion-back-button>
+      </ion-buttons>
+
+      <ng-container *ngIf="parentArea; else noArea">
+          <ion-title>{{ parentArea.name }}</ion-title>
+      </ng-container>
+      <ng-template #noArea>
+        <ion-title>{{ title }}</ion-title>
+      </ng-template>
+
+
+      <ion-buttons slot="end">
+        <ion-button  (clic)="clicValid()" size="large">
```

Comment:
Change `(clic)` to `(click)`. Also rename the function to something more explicit, like optionsClicked

Reply:
What is `(click). ?`